ext/snmp/snmp.stub.php

```diff
@@ -87,32 +87,32 @@ class SNMP
 
     public function __construct(int $version, string $hostname, string $community, int $timeout = -1, int $retries = -1) {}
 
-    /** @return bool */
-    public function close() {}
+    /** @tentative-return-type */
+    public function close(): bool {}
 
-    /** @return bool */
+    /** @tentative-return-type */
     public function setSecurity(
         string $securityLevel, string $authProtocol = "", string $authPassphrase = "",
         string $privacyProtocol = "", string $privacyPassphrase = "",
-        string $contextName = "", string $contextEngineId = "") {}
+        string $contextName = "", string $contextEngineId = ""): bool {}
 
-    /** @return array|bool */
-    public function get(array|string $objectId, bool $preserveKeys = false) {}
+    /** @tentative-return-type */
+    public function get(array|string $objectId, bool $preserveKeys = false): array|string|false {}
```

Comment:
The manual writes: "Returns SNMP objects requested as string or array depending on objectId type or false on error." This is also something I'm not able to verify. :(